2022/Q2/artifacts/pos-model/PoS-model.md

```diff
@@ -261,8 +261,8 @@ func withdraw(validator_address, delegator_address)
     //COMMENT: is the amount slashed here the same than the one slashed when evidence is found? This is an important point
     var amount_after_slashing = unbond.amount
     forall (slash in slashes[validator_address] s.t. start <= slash.epoch && slash.epoch <= end)
-      amount_after_slashing *= (10000 - slash.rate) / 10000)
-    balance[validator_address] += amount_after_slashing
+      amount_after_slashing += unbond.amount * slash.rate
```

Comment:
```suggestion
      amount_after_slashing += amount * slash.rate
```

Reply:
Thanks for this Tomas. I have added a couple of changes, but it is not related to your changes: we were using unbond.amount, when we should have used simply amount.